app/instance-initializers/ember-a11y.js

```diff
@@ -1,10 +1,12 @@
 // This initializer exists only to make sure that the following
 // imports happen before the app boots.
 import Ember from 'ember';
-import { registerKeywords } from 'ember-a11y/ember-internals';
 import pivotRouteIdentifier from 'ember-a11y/utils/pivot-route-identifier';
 
+//{{htmlbars}}
+import { registerKeywords } from 'ember-a11y/ember-internals';
```

Comment:
right, initially I had it exactly like that but Glimmer version of `ember-internals` is a module with an empty function so @nathanhammond proposed to drop it altogether for Glimmer